unpackaged/config/rd2_post_config/flexipages/NPSP_Recurring_Donation.flexipage

```diff
@@ -0,0 +1,113 @@
+        </componentInstances>
+        <name>facet-1ac173b3-bbf9-42df-b2cf-d9c3e4adb6dc</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>relatedListComponentOverride</name>
+                <value>NONE</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>rowsToDisplay</name>
+                <value>10</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>showActionBar</name>
+                <value>true</value>
+            </componentInstanceProperties>
+            <componentName>force:relatedListContainer</componentName>
+        </componentInstances>
+        <name>facet-df729d2c-0f85-4d90-ac8b-f44fa7f5078b</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>body</name>
+                <value>facet-1ac173b3-bbf9-42df-b2cf-d9c3e4adb6dc</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>title</name>
+                <value>Standard.Tab.detail</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tab</componentName>
+        </componentInstances>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>active</name>
+                <value>false</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>body</name>
+                <value>facet-df729d2c-0f85-4d90-ac8b-f44fa7f5078b</value>
+            </componentInstanceProperties>
+            <componentInstanceProperties>
+                <name>title</name>
+                <value>Standard.Tab.relatedLists</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tab</componentName>
+        </componentInstances>
+        <name>facet-7003451d-d97d-44c2-b74f-b56f89537a36</name>
+        <type>Facet</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>tabs</name>
+                <value>facet-7003451d-d97d-44c2-b74f-b56f89537a36</value>
+            </componentInstanceProperties>
+            <componentName>flexipage:tabset</componentName>
+        </componentInstances>
+        <name>main</name>
+        <type>Region</type>
+    </flexiPageRegions>
+    <flexiPageRegions>
+        <componentInstances>
+            <componentInstanceProperties>
+                <name>displayNum</name>
+                <value>12</value>
+            </componentInstanceProperties>
+            <componentName>%%%NAMESPACE_OR_C%%%rdScheduleVisualizer</componentName>
```

Comment:
@davidmreed In a non-namespaced org, the above token is getting replaced a value of "c:c". The error I get in deployment is `Update of FlexiPage NPSP_Recurring_Donation: Error: We couldn't retrieve the design time component information for component c:crdScheduleVisualizer`. Seems strange to be honest.

Reply:
Though, since Reede didn't see that error when he ran this task, maybe it only happens when the task is run after the flexipage is deployed the first time. Now that would be weird.

Reply:
I am seeing that error now.

Reply:
Interesting. Clearly the second time is the charm, which really makes no sense. Any thoughts on what might be happening here @davidmreed?

Reply:
I tried the install a 2nd time on a fresh org to see if it was intermittent, and it showed up again.  I think it is likely that I just missed the error on my initial test.

Reply:
I'll dig into this as soon as we get the release out.